o4/o4.py

```diff
@@ -16,7 +16,10 @@
 
 Option:
   sync          Sync/verify <path>.
-  status        Similar to git status, verifies the files in <path> against fstat records.
+  status        Similar to git status, verifies the files in <path> against fstat records. By
+                default only files that are supposed to exist are checked, to check all files,
+                provide -f. For a faster check from 80% of current changelist,
+                add -q (recommended).
```

Comment:
-q makes total sense for core. It's still hundreds of thousands of files and you don't spend time checking things that *never* change.